### ISSUE TITLE
[Screen Readers-Ask HR Virtual Assistant]: Screen readers(NVDA/JAWS) are narrating the suggestion invoked by the user multiple times upon invoking the Suggested Questionnaire button present in the “Ask HR Virtual Assistant” chat window.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#4138]
**User Experience:**
Screen reader users are getting impacted if  Screen readers(NVDA/JAWS) are narrating the suggestion invoked by the user multiple times upon invoking the Suggested Questionnaire button present in the “Ask HR Virtual Assistant” chat window. 

Note: User credentials should NOT be included in the bug.

**Repro Steps:​**
1. Enable Screen readers (NVDA/JAWS) and Open the above URL in the Chrome/Edge Browser using valid credentials. 
2. 'AskHR Virtual Assistant' Home page gets opened.
3. Navigate to the list items suggestions using down arrow key and select Benefits verify the screen readers(NVDA/JAWS) behavior upon invoking the suggestion questionnaire button present in the “Ask HR Virtual Assistant” chat window. 

**Actual Result:**
Screen readers(NVDA/JAWS) are narrating the suggestion invoked by the user multiple times upon invoking the Suggested Questionnaire button present in the “Ask HR Virtual Assistant” chat window.
Ex: NVDA/JAWS narrating as "Suggested Actions Container: Is empty………………………………complementary landmark form landmark form landmark message input box edit type your message blank Suggested Actions Container: Has content  Press Alt+ Shift + A to select Laya Pension(Mercer)Well being program ESPP other benefits None of these/ask a different question "

**Expected Result:**
Screen readers(NVDA/JAWS) shouldn't narrate the suggestion invoked by the user multiple times upon invoking the Suggested Questionnaire button present in the “Ask HR Virtual Assistant” chat window.

Ex: NVDA/JAWS should narrate as "Select an option to continue this conversation Suggested Actions Container: Has content  Press Alt+ Shift + A to select 6 list items button Laya, button Pension(Mercer), button Well being program, button  ESPP, button  other benefits, button None of these/ask a different question"

**MAS reference:**
[WCAG4.2.1 – Object Information](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7bEEE97BBD-9990-48A0-8B03-6EA775581AA8%7d&file=MAS%204.2.1%20%E2%80%93%20Object%20Information.docx&wdLOR=cBC823FF2-F30B-4029-A49F-90FCC0AB9FEB&action=default&mobileredirect=true)

**Reference Links:**

- [Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-poyad%40microsoft.com%7cc11478591b5a474f6d9d08d7199726c0%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637006009300301864&sdata=ZTWYamT1yE7yko5QU35ZJLf/5ZydtDeGAKrQGUiugJs%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this [Link](https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx). “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
- Please reach out to [C&AI Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cv-dhti%40microsoft.com%7cf90880a914be401871f108d763233116%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637086876782361405&sdata=/ULS%2BbeqMjysCoiWHvFgAbSnuFJqxO3wtpHmlidPjbU%3D&reserved=0) for any process related queries.

**Test Environment:** 
OS: Windows 10 version 21H1 (OS Build 19043.1110)
Browser: New Edge Version 98.0.1100.3 (Official build) (64-bit)/Chrome Version 92.0.4515.107 (Official Build) (64-bit)
Tools: NVDA 2021.1/JAWS 2021.2105.53 ILM
URL: [AskHR Virtual Assistant (azureedge.net)](https://askhrvauat.azureedge.net/?botorigin=hrweb)

**Comment by KattaBhargav** [comment]
#A11yMAS;#A11ySev2;#HCL;#WCAG4.3.1;#Accessibility;#Win10-Edge(Chromium);#BotFrameworkWebChat-Aug21;#Benchmark;#CSEO:BUG8007103;#Firstparty:CSEO;#Regressed:02-09-22;#Regression;

**Comment by KattaBhargav** [comment]
Bug copied from CSEO team Bug id:-[BUG 8007103](https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/8007103)

**Comment by compulim** [comment]
@KattaBhargav could you grant my alias to the UAT site of AskHR so I can repro it?

**Comment by gabog** [comment]
hey @KattaBhargav, were you able to grant access to William to the site?

**Comment by gabog** [comment]
Closing this one due to inactivity
